tableauserverclient/server/image_request_options.py

```diff
@@ -0,0 +1,20 @@
+from .request_options_base import RequestOptionsBase
+
+
+class ImageRequestOptions(RequestOptionsBase):
+    class Resolution:
+        High = 'high'
```

Comment:
ok. can you please add a comment here that says if you don't specify 'high' that the underlying REST API will default to standard?